packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -85,10 +85,10 @@ const String _flutterRenderingLibrary = 'package:flutter/rendering.dart';
 /// different parents. The scene must be explicitly recomposited after such
 /// changes are made; the layer tree does not maintain its own dirty state.
 ///
-/// To composite the tree, create a [SceneBuilder] object, pass it to the
-/// root [Layer] object's [addToScene] method, and then call
-/// [SceneBuilder.build] to obtain a [Scene]. A [Scene] can then be painted
-/// using [dart:ui.FlutterView.render].
+/// To composite the tree, create a [SceneBuilder] object using
+/// [RendererBinding.createSceneBuilder], pass it to the root [Layer] object's
```

Comment:
Maybe also update the dart:ui documentation on SceneBuilder, PictureRecorder, and PictureRecorder to at least cross-link to the create* methods on RenderBinding? If I need to create any of those objects I would look there first and probably totally miss that I should be going through the bindings.

Reply:
will do

Reply:
https://github.com/flutter/engine/pull/52374